code-challenges.js

```diff
@@ -18,10 +18,45 @@ const fruit3 = "cherry"
 const fruit4 = "kiwi"
 // Expected outcome: "cherry"
 
+
+
+
+
+
+
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Describe your process:
 
+// I need to first think of what structure I will use. I am thinking a function w/ conditionals, that way I can get different answers depending on the degree its at; below, at, or above boiling point.
+
+// I also need to decide on what declaration I will give our variable (const), most likely i'll use "boilingPoint" to keep it on topic.
+
+// For the parameter I will be using "temp" to also keep this relative. 
+
+// In the curly braces/code block is where my conditional will go in for below, at, or above boiling point.
+
+// I am going to put string interpolation so it will match the expected outputs layout of the parameter and the "is below boiling point" etc.
+
+// Next I have to give it a shot and call on the function with console.log
+
+const boilingPoint = (temp) => {
+    if (temp < 212)
+        return `${temp} F° is below boiling point`
+    else if (temp > 212)
+        return `${temp} F° is above boiling point`
+    else if (temp == 212) 
+        return `${temp} F° is at boiling point`
+    else 
+       return '---.--'
+}
+console.log (boilingPoint ())
+
+//I had to play around in line 42 and finally made it == (loose equality) so my else statement would return '---.--' so it would look like a cooking thermometer before checking a temperature. Leaving it = weirdly gave me '212 F° is at boiling point'. Making it a === (strict equality) is also now working, it wasn't working, I forgot in what circumstance it wasn't working... weird 
```

Comment:
I would always recommend using strict equality as this is super specific and does not allow Javascript to manipulate the data type on you.  Also, be sure to add multiple console logs that invoke the function, one for each test variable.  Nice work on the logic and adding the degree symbol!